#!/usr/bin/env python
PACKAGE = "rcbigctl"

from dynamic_reconfigure.parameter_generator_catkin import *
from math import pi

gen = ParameterGenerator()

gen.add("MaxX", double_t, 0, "Max Linear X Speed (m/s)",  1, 0, 10)
gen.add("MaxY", double_t, 0, "Max Linear Y Speed (m/s)",  1, 0, 10)
gen.add("MaxW", double_t, 0, "Max Angular Speed (rad/s)",  2 * pi, 0, 3 * 2 * pi)

exit(gen.generate(PACKAGE, "teleop", "Teleop"))